docs/source/api_reference.rst

```diff
@@ -21,6 +21,17 @@ Defined in ``xtensor-io/xnpz.hpp``
 .. doxygenfunction:: xt::dump_npz
    :project: xtensor-io
 
+HDF5 files
+----------
+
+Defined in ``xtensor-io/xhighfive.hpp``
+
+.. doxygenfunction:: xt::dump_hdf5
+   :project: xtensor-io
+
+.. doxygenfunction:: xt::load_hdf5
+   :project: xtensor-io
+
```

Comment:
add other signatures

Reply:
Will do.

Reply:
Actually, there is only one signature with an all accepting template. So this should be correct?